profiling-ffi/src/exporter.rs

```
@@ -192,6 +192,20 @@ fn ddog_prof_exporter_new_impl(
     )
 }
 
+/// Sets the timeout for the exporte's timeout.
+/// # Arguments
+/// * `exporter` - ProfileExporter instance.
+/// * `timeout_ms` - timeout in milliseconds.
+#[no_mangle]
+pub unsafe extern "C" fn ddog_prof_Exporter_set_timeout(
+    exporter: Option<&mut ProfileExporter>,
+    timeout_ms: u64,
+) {
+    if let Some(ptr) = exporter {
```

Should we return an error in this case?